app/posts/register-trainee-teachers/2022-02-10-autocomplete-analytics.md

```diff
@@ -0,0 +1,239 @@
+
+### Sample of recent searches
+
+{{ govukTable({
+  caption: "Some example searches from our autocomplete analytics",
+  captionClasses: "govuk-table__caption--m",
+  firstCellIsHeader: false,
+  head: [
+    {
+      text: "Failed search"
+    },
+    {
+      text: "Successful search"
+    },
+    {
+      text: "Match"
+    },
+    {
+      text: "Identified issue"
+    }
+  ],
+  rows: [
+    [
+      {
+        text: "134798"
+      },
+      {
+        text: "(not set)"
+      },
+      {
+        text: "(not set)"
+      },
+      {
+        text: "Missing entry?"
+      }
+    ],
+    [
+      {
+        text: "jew"
+      },
+      {
+        text: "silver"
+      },
+      {
+        text: "Silversmithing and Goldsmithing"
+      },
+      {
+        text: "Missing entry or synonym"
+      }
+    ],
+    [
+      {
+        text: "teachi,learn,mentor"
+      },
+      {
+        text: "educa"
+      },
+      {
+        text: "Education Studies"
+      },
+      {
+        text: "Synonym"
+      }
+    ],
+    [
+      {
+        text: "birmimgh,uce,centreal ,central en,birim"
+      },
+      {
+        text: "birmin"
+      },
+      {
+        text: "The University of Birmingham"
+      },
+      {
+        text: "Typo"
+      }
+    ],
+    [
+      {
+        text: "PE"
+      },
+      {
+        text: "Phy"
+      },
+      {
+        text: "Physical education"
+      },
+      {
+        text: "Abbreviation"
+      }
+    ],
+    [
+      {
+        text: "uk ope"
+      },
+      {
+        text: "Open"
+      },
+      {
+        text: "The Open University"
+      },
+      {
+        text: "Synonym"
+      }
+    ],
+    [
+      {
+        text: "bs"
+      },
+      {
+        text: "bus"
+      },
+      {
+        text: "Business studies"
+      },
+      {
+        text: "Abbreviation"
+      }
+    ]
+  ]
+}) }}
+
+## What we’ve found and how we can improve our autocompletes
+
+This data can not be easily quantified – it needs manual analysis to pick out common themes and issues. But there are place where we can see ways to improve the autocomplete for our users
+
+### Typos
+
+A significant number of ‘failed’ searches are typos. These are to be expected and do not necessarily indicate a problem with the autocomplete,  especially where the user clearly self-corrected and found what they were looking for. However if we see some repeatedly we could consider adding those as synonyms.
+
+### Synonyms
+
+In many cases the word the user tries is not in the list, but there is a word that means the same thing. A good example from the table above is a user searching for `teachi`, `learn`, and `mentor` – for which the user ultimately picked `Education studies`. We could consider adding `Teaching` as a synonym for `Education studies`.
```

Comment:
```suggestion
In many cases the word the user tries is not in the list, but there is a word that means the same thing. A good example from the table above is a user searching for `teachi`, `learn`, and `mentor` — for which the user ultimately picked `Education studies`. We could consider adding `Teaching` as a synonym for `Education studies`.
```

I also find the 'code' syntax a bit hard to read here, and I'm not sure its semantically correct.